hack/verify-gofmt.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck source=/dev/null
+source "$(dirname "$0")/utils.sh"
+# cd to the root path
+cd_root_path
+
+# check for gofmt diffs
+diff=$(git ls-files | grep "\.go" | grep -v "\/vendor" | xargs gofmt -s -d 2>&1)
```

Comment:
Is there any benefit to the `gofmt -s` check? is this standard style across most projects? I'd prefer to just use the standard go tool chain and require `go fmt`.

The reason i'm questioning this is that some editors make it difficult to run multiple formatters. For instance, running `goimports` and `gofmt -s` is annoying in vscode because they require different arguments. Goimports doesn't format code and gofmt doesn't manage imports.

If this is standard across kubernetes though then I'm ok to keep it.

Reply:
k/k does use `go fmt` in `./hack/verify-gofmt.sh`

Reply:
ah yes, I see it. Ok, this is fine